### DISCUSSION TITLE
Please Add New extension of Best Payment Getaway of India.

### DISCUSSION THREAD

**Discussion by ghost** [Paymenter/Paymenter#521] [Ideas]
Today, I would like to request the payment team to introduce "PhonePa" gateway in their payment extensions because more than 46% of Indian users prefer "PhonePa" payment gateway and this UPI gateway is more trusted, secure and popular than other Indian UPI gateways.

why are all businesses switching to PhonePe payment gateway.
1. It's easy to register. 
2. Fast services.
3. Multi payment option. 
4. 0% transaction fees. 
5. 100% free service.
6. More secure & private.
7. More trusted for client.
8. Very popular and much users.

I will request that you bring this payment gateway as soon as possible. I have full trust in your team. God bless to you guys.

**Comment by zvikasdongre** [comment]
Maybe see if the [ManualMultiPay](https://market.paymenter.org/extensions/42/ManualMultiPay) extension works for you.

But beware that it does not automatically handle invoicing or payment confirmation:
> Please note that this extension does not handle payment confirmation or invoice marking. **After a user makes a payment, you must manually confirm that the payment has been made and mark the invoice as paid.** This extension is designed to facilitate the payment process, but the confirmation and invoice marking processes are still your responsibility.
